src/components/common/BusinessContactInfo.vue

```diff
@@ -176,7 +176,7 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
   // Rules for template
   readonly Rules = Rules
 
-  contact = this.initialValue
+  contact = { ...EmptyContactPoint } as ContactPointIF
```

Comment:
This is a bug fix. Previously, this component did not react if the contact point changed.